zip-1014.rst

```diff
@@ -0,0 +1,424 @@
+
+
+Funding Target and Volatility Reserve
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Each Dev Fund slice has a Funding Target, initially US $700,000 for each
+slice. At the end of each calendar month, the fair market value of the Dev
+Fund ZEC received during that month will be computed, and the excess over
+the Funding Target will be put into a dedicated Volatility Reserve account
+by the funds' recipient.
+
+Funds may be withdrawn from the Volatility Reserve account only by that same
+party, in months where the aforementioned monthly ZEC value falls short of
+the Funding Target, and only to the extent needed to cover that shortfall.
+
+The Volatility Reserve may be kept as ZEC, or sold and held as fiat currency
+or investments (whose profits will remain in the Volatility Reserve).
+
+The Funding Target may be changed only by unanimous agreement of ZF, ECC, and
+the majority vote of the Community Panel. (In case of excessive 
+accumulation of reserves, the community MAY condition an increase of the
+Funding Target on the redirection of some of the reserves to a different
+entity, miners or an airdrop.)
+
+Dev Fund ZEC that has been received, not placed in the Volatility Reserve,
+and has not yet been used or disbursed, will be kept by the corresponding
+party (as ZEC, or sold and invested) for later use under the terms of the
+corresponding slice.
+
+Irrevocable obligations to the above must be made by the recipients (e.g.,
+using their Operating Agreements or by receiving the slice as Restricted
+Funds).
+
+
+Transparency and Accountability
+-------------------------------
+
+Obligations
+~~~~~~~~~~~
+
+ECC, ZF, and Major Grant recipients (during and leading to their award period)
+SHALL all accept the following obligations:
+
+Ongoing public reporting requirements:
+
+* Quarterly reports, detailing future plans, execution on previous plans, and
+  finances (balances, and spending broken down by major categories).
+* Monthly developer calls, or a brief report, on recent and forthcoming tasks.
+  (Developer calls may be shared.)
+* Annual detailed review of the organization performance and future plans.
+* Annual audited financial report (IRS Form 990, or substantially similar
+  information).
+
+These reports may be either organization-wide, or restricted to the income,
+expenses, and work associated with the receipt of Dev Fund.
+
+It is expected that ECC, ZF, and Major Grant recipients will be focused
+primarily (in their attention and resources) on Zcash. Thus, they must
+promptly disclose:
+
+* Any major activity they perform (even if not supported by the Dev Fund) that
+  is not in the interest of the general Zcash ecosystem.
+* Any conflict of interest with the general success of the Zcash ecosystem.
+
+ECC, ZF, and grant recipients must promptly disclose any security or privacy
+risks that may affect users of Zcash (by responsible disclosure under
+confidence to the pertinent developers, where applicable).
```

Comment:
Is this flexible enough to permit extended periods of secrecy, e.g. as occurred for the BCTV14 flaw?